website/content/en/preview/AWS/provisioning.md

```diff
@@ -6,6 +6,40 @@ description: >
   Learn AWS-specific Karpenter provisioning settings
 ---
 
+Provisioner settings specific to Karpenter for the AWS cloud provider are described here.
+The following example shows optional and required settings for a Karpenter provisioner for AWS:
+
+
+```yaml
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  requirements:                      
+    - key: karpenter.sh/capacity-type         # optional, on-demand by default; spot if both are listed
+      operator: In
+      values: ["spot"]
+  limits:   
+    resources:
+      cpu: 1000                               # optional, recommended to limit total cluster CPU
+      memory: 1000Gi                          # optional, recommended to limit total cluster memory
+  provider:
+    subnetSelector:                           # required
+      karpenter.sh/discovery: ${CLUSTER_NAME}
+    securityGroupSelector:                    # required

```

Comment:
this is not required when using the launchTemplate field.

Maybe we can note it:

`# required when not using [launchTemplate](#LaunchTemplate)`

Reply:
Okay. Done.